### PR TITLE
Add Metrics to Encrypt Calls

### PR DESCRIPTION
**Pull request by adamisntdead**
# Why

Currently we have no way of knowing how many fields are getting encrypted using the Evervault

# How

Adds a call to the metrics endpoint with a `Data Encrypted` event. Also stops `metrics` and `ca` subdomains from going through Relay

### CODE REVIEW

File: lib/core/http.js [hunk 1]
```diff
@@ -61,5 +61,9 @@ module.exports = (config) => {
     });
   }
 
-  return { getCageKey, runCage, getCert };
+  const reportMetric = () => {
+    return post(config.metricsUrl, '', { event: 'Data Encrypted' })
```
Comment (eoinboylan) [inline lib/core/http.js:65]:
```suggestion
    return post(config.metricsUrl, { event: 'Data Encrypted' })
```

Comment (eoinboylan) [reply lib/core/http.js:65]: Is this the wrong way around and should be above?

**Review by eoinboylan — CHANGES_REQUESTED** [review]

File: lib/core/http.js (hunk 1 above)
Comment (adamisntdead) [reply lib/core/http.js:65]: Yes, it's the opposite in the client side SDK, will commit the change

**Comment by adamisntdead** [comment]
(This won't be ready to merge until I actually test that the metrics are being posted, since the package will automatically be deployed on push)

**Comment by adamisntdead** [comment]
Also if a user encrypts strings in bulk, these metrics will fail

**Review by eoinboylan — APPROVED** [review]

**Comment by adamisntdead** [comment]
Closing for now, can reopen later if needed

**Pull request closed without merging**